packages/hash/frontend/src/pages/new/entity.page.tsx

```diff
@@ -21,19 +19,21 @@ const Page: NextPageWithLayout = () => {
     loadingTypeSystem || !queryEntityId
       ? null
       : validateVersionedUri(queryEntityId);
-  const { authenticatedUser, loading: authenticatedUserLoading } =
-    useAuthenticatedUser(undefined, true);
+  const { activeWorkspace } = useContext(WorkspaceContext);
   const shouldBeCreatingEntity = entityTypeId?.type === "Ok";
   const [creatingEntity, setCreatingEntity] = useState(shouldBeCreatingEntity);
 
-  const entityIdInvalid = !!queryEntityId && entityTypeId?.type === "Err";
+  const entityTypeInvalid = !!queryEntityId && entityTypeId?.type === "Err";
 
   if (shouldBeCreatingEntity && !creatingEntity) {
     setCreatingEntity(true);
-  } else if (entityIdInvalid && creatingEntity) {
+  } else if (entityTypeInvalid && creatingEntity) {
     setCreatingEntity(false);
   }
 
+  // @todo better way to force login when using active workspace?
+  useLoggedInUser();
+
```

Comment:
I think this is fine.

Reply:
This is going to be replaced with server side redirects.